packages/node/src/parsers.ts

```diff
@@ -9,6 +8,8 @@ import { NodeOptions } from './types';
 const DEFAULT_LINES_OF_CONTEXT: number = 7;
 const FILE_CONTENT_CACHE = new LRUMap<string, string | null>(100);
 
+export type ReadFileFn = (path: string, callback: (err: Error | null, data: Buffer) => void) => void;
```

Comment:
```suggestion
export type ReadFileFn = (path: string, callback: (err: Error | null, data: string) => void) => void;
```

Cant we use `fs.readFile` type directly though? We are in `node` package here, not `core` so it should be fine.

Reply:
I'll have another look but it seemed like `fs.readFile` is not a type that's exported.

Reply:
It looks like you call `toString()` on `data` so a assume it is a `Buffer` rather than `string`?

```ts
        readFile(filename, (err: Error | null, data: Buffer) => {
          const content = err ? null : data.toString();
```